WalletWasabi.Fluent/Controls/LoadingControl.axaml

```diff
@@ -0,0 +1,22 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:c="using:WalletWasabi.Fluent.Controls">
+
+  <Style Selector="c|LoadingControl">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <DockPanel LastChildFill="True" HorizontalAlignment="Center" VerticalAlignment="Center" MaxWidth="500">
+          <TextBlock Text="{TemplateBinding StatusText}" Opacity="0.6" DockPanel.Dock="Bottom" Margin="0 10 0 0" />
+          <ProgressBar Value="{TemplateBinding Percent}" IsIndeterminate="{Binding !$self.Value}" DockPanel.Dock="Bottom" />
+          <TextBlock Text="This process may take a long time depending on the size of your wallet. Grab a coffee or watch a movie."
```

Comment:
IMO we should rephrase this because it feels it is going to take forever to load the wallet, which might be true if the wallet is too big. I would simply say:

`This process may take some time depending on the size of your wallet.`

Reply:
ACK

Reply:
Thank you for considering that, but IMO I think we better remove this part `Grab a coffee or watch a movie.`